pkg/reconciler/events/cloudauditlog/filterbuilder.go

```diff
@@ -0,0 +1,52 @@
+package cloudauditlog
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	keyPrefix   = "protoPayload"
+	methodKey   = keyPrefix + ".methodName"
+	serviceKey  = keyPrefix + ".serviceName"
+	resourceKey = keyPrefix + ".resourceName"
+	typeKey     = keyPrefix + ".\x22@type\x22"
+	typeValue   = "type.googleapis.com/google.cloud.audit.AuditLog"
+)
+
+type FilterBuilder struct {
+	serviceName  string
+	methodName   string
+	resourceName string
+}
+
+func (fb *FilterBuilder) SetServiceName(serviceName string) {
+	fb.serviceName = serviceName
+}
+
+func (fb *FilterBuilder) SetMethodName(methodName string) {
+	fb.methodName = methodName
+}
+
+func (fb *FilterBuilder) SetResourceName(resourceName string) {
+	fb.resourceName = resourceName
+}
+
+func (fb *FilterBuilder) GetFilterQuery() string {
+	var filters []string
+	if fb.methodName != "" {
+		filters = append(filters, fmt.Sprintf("%s=%q", methodKey, fb.methodName))
```

Comment:
Filter values should probably be quoted.

Reply:
%q	a double-quoted string safely escaped with Go syntax 
FYI: https://golang.org/pkg/fmt/
The filter values already got double-quoted